src/plugins/privacy/plugin.py

```diff
@@ -467,7 +467,7 @@ def _patch(self, initiator: File):
 
             # Replace callback
             def replace(match: Match):
-                value = match.group(1)
+                value = match.group("url")
```

Comment:
It seems the issue was that `replace` was called on the `self.assets_expr_map[extension]` entry in the parent method, which we've changed so positional groups didn't always give reliable results.

Reply:
Perfect, thanks for investigating and fixing!